modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/GridCacheDatabaseSharedManager.java

```diff
@@ -190,12 +191,15 @@
 @SuppressWarnings({"unchecked", "NonPrivateFieldAccessedInSynchronizedContext"})
 public class GridCacheDatabaseSharedManager extends IgniteCacheDatabaseSharedManager {
     /** */
+    @SystemProperty(value = "Skip IO sync on checkpoint.")
```

Comment:
"Sets the flag controlling if the I/O sync needs to be skipped on a checkpoint. Defaults to`false`"

Reply:
Fixed.